.github/workflows/test.yml

```diff
@@ -5,8 +5,18 @@ on:
   pull_request:
 
 jobs:
-  test74:
-    runs-on: ubuntu-latest
+  phpunit:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
```

Comment:
By using a matrix. You only need to define the job once, and let it run with the defined values. In this case multiple PHP versions.